File: backend-csharp/Models/DonViVanChuyen.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class DonViVanChuyen
{
    public long Id { get; set; }

    public string TenDonVi { get; set; } = null!;

    public string Ma { get; set; } = null!;

    public string? LogoUrl { get; set; }

    public string? ApiKey { get; set; }

    public string? ApiEndpoint { get; set; }

    public decimal PhiCoBan { get; set; }

    public string? ThoiGianDuKien { get; set; }

    public string TrangThai { get; set; } = null!;

    public virtual ICollection<DonHang> DonHangs { get; set; } = new List<DonHang>();
}

File: backend/models/DonViVanChuyen.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const DonViVanChuyen = sequelize.define(
  "DonViVanChuyen",
  {
    id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
    ten_don_vi: { type: DataTypes.STRING(100) },
    ma: { type: DataTypes.STRING(20) },
    logo_url: { type: DataTypes.STRING(255) },
    api_key: { type: DataTypes.STRING(255) },
    api_endpoint: { type: DataTypes.STRING(255) },
    phi_co_ban: { type: DataTypes.DECIMAL(15, 2) },
    thoi_gian_du_kien: { type: DataTypes.STRING(50) },
    trang_thai: { type: DataTypes.STRING(20), defaultValue: "active" },
  },
  { tableName: "DonViVanChuyen", timestamps: false },
);

module.exports = DonViVanChuyen;
